skore/tests/unit/sklearn/test_cross_validation.py

```diff
@@ -337,7 +337,8 @@ def _check_results_single_metric(report, metric, expected_n_splits, expected_nb_
 
     # check that the columns contains the expected split names
     split_names = result.columns.get_level_values(1).unique()
-    expected_split_names = [f"Split #{i}" for i in range(expected_n_splits)]
+    # expected_split_names = [f"Split #{i}" for i in range(expected_n_splits)]
```

Comment:
Actually, instead of changing the test here, let's change the value of aggregate:

```python
result = getattr(report.metrics, metric)(aggregate=None)
```

We have other location where we already check the behaviour of requesting the mean and std